### PR TITLE
matrix updates

### PR DESCRIPTION
**Pull request by naugtur**
- added a case found by @kriskowal 
- added Bun to the list of bundlers (with some annoying quirks in how it runs, sorry)

CI is failing because SES wants newer yarn. That's gotta be a separate PR.

### CODE REVIEW

**Review by kriskowal — APPROVED** [review]

File: matrix/results/bun.json [hunk 1]
```diff
@@ -0,0 +1,191 @@
+{
+  "name": "bun",
+  "bin": "node",
+  "args": [
+    "/home/naugtur/repo/endo-e2e-tests/matrix/cases/import-cjs/dist/bun.mjs"
```
Comment (kriskowal) [inline matrix/results/bun.json]: Non-portable fully-qualified path.

File: matrix/results/node-v20.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
+{
+  "name": "node-v20",
+  "bin": "node",
+  "args": [
+    "/home/naugtur/repo/endo-e2e-tests/matrix/cases/import-cjs/index.mjs"
```
Comment (kriskowal) [inline matrix/results/node-v20.json]: ditto

Comment (naugtur) [reply matrix/results/node-v20.json]: lol, how did taht end up there?

File: matrix/results/bun.json (hunk 1 above)
Comment (naugtur) [reply matrix/results/bun.json]: these are just documenting what happened during the run. But I turned them into relative paths to get rid of this.

**Pull request merged**